state/execution_test.go

```diff
@@ -1123,6 +1123,110 @@ func stripSignatures(ec *types.ExtendedCommit) {
 	}
 }
 
+// TestCreateProposalBlockReapsMoreThanMaxDataBytes verifies that CreateProposalBlock
+// reaps 125% of maxDataBytes to provide buffer for PrepareProposal
+func TestCreateProposalBlockReapsMoreThanMaxDataBytes(t *testing.T) {
```

Comment:
this test adds more tech debt than value and this is already tested in the mempool therefore I think we should remove